tests/input/examples/schema_definition-array-2.yaml

```diff
@@ -26,7 +26,7 @@ classes:
     implements:
       - linkml:NDArray
     annotations:
-      dimensions: 1
+      dimensions_info: "1"
```

Comment:
Should I reformat these to use:
```yaml
        array_info:
          exact_dimensions: 1
```

Reply:
For this TimestampSeries class, I'm actually not sure it would be allowed to put `array_info` at the class definition level, but maybe for the other ones?

Reply:
unfortunately that doesn't quite work because the range of `dimensions_info` is a collection of `dimension_expression`s, so these have to be at a different level

e.g. this is invalid yaml:

```yaml
dimensions_info: 3
  x:
  y:
  z:
```